### PR TITLE
add my stake section

### PR DESCRIPTION
**Pull request by AlexZhenWang**
https://user-images.githubusercontent.com/14008461/104980678-0615ec80-5a6c-11eb-8e3b-30b60da31477.mov

Current UI:
image

TODO:
- [x] Group the bond information and the nominations related to the bond to one section. 
- [x] Add tooltips for concepts on my stake page
- [x] Update license (added a ticket)
- [ ] Add spinner

### CODE REVIEW

File: packages/app-staking/src/Overview/MyStake/utils.tsx [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+          resolve();
+        })
+    )
+  );
+  return nominates;
+}
+
+// Reference: https://github.com/cennznet/cennznet/wiki/Validator-Guide#rewards
+// TODO: is 0.8 dynamic?
+const stakersPayRate = new BN(0.8); // 80% to stakers
```
Comment (AlexZhenWang) [inline packages/app-staking/src/Overview/MyStake/utils.tsx:189]: Hi guy, may I double-check, is this stakersPayRate dynamic, if it is, is there any interface I get can this value(I can't find any interface for this)?

Comment (holygits) [reply packages/app-staking/src/Overview/MyStake/utils.tsx:189]: we plan to build an RPC to return the correct values, in next node update. This value is `100% - validator's commission %` e.g if validator has 10% commission, stakerRate would be 90%

Comment (AlexZhenWang) [reply packages/app-staking/src/Overview/MyStake/utils.tsx:189]: Thank you @holygits. I will update the code.

**Comment by alexsednz** [comment]
>  Screen.Recording.2021-01-19.at.3.34.55.PM.mov

As suggested today, if you could box a bond with the corresponding "Nominating" rows below it using dotted lines or label boundaries, the display will improve visually.

File: packages/app-staking/src/Overview/MyStake/index.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
+// Copyright 2017-2020 @polkadot/app-staking authors & contributors
```
Comment (alexsednz) [inline packages/app-staking/src/Overview/MyStake/index.tsx:1]: The copyright should mention Centrality too.

**Comment by AlexZhenWang** [comment]
> > Screen.Recording.2021-01-19.at.3.34.55.PM.mov
> 
> As suggested today, if you could box a bond with the corresponding "Nominating" rows below it using dotted lines or label boundaries, the display will improve visually.

thank you for the reminder mate @alexsednz     Working on it now

File: packages/app-staking/src/Overview/MyStake/index.tsx (hunk 2 above)
Comment (AlexZhenWang) [reply packages/app-staking/src/Overview/MyStake/index.tsx:1]: Sure. Will update it

**Comment by AlexZhenWang** [comment]
Updated the UI based on the feedback:
Group bond and nominations together
image

File: packages/app-staking/src/Overview/MyStake/index.tsx (hunk 2 above)
Comment (AlexZhenWang) [reply packages/app-staking/src/Overview/MyStake/index.tsx:1]: Updated

**Comment by AlexZhenWang** [comment]
Feedback from the team on 21/01/2021: 

- [ ] Group the bond information and the nominations related to the bond to one section. 
- [ ] Add tooltips for all concept on my stake page

**Comment by AlexZhenWang** [comment]
Feedback from the team on 21/01/2021: 

1. Group the bond information and the nominations related to the bond to one section. 
2. Add tooltips for all concept on my stake page

**Comment by AlexZhenWang** [comment]
Add tooltips
```
Stake: 'This is the primary account that holds the funds and has a portion bonded for participation',

Controller: 'This is used to control the operation of the validator or nominator, switching between validating, nominating and idle (It only needs enough funds to send transactions when actions are taken)',

Reward Destination: 'This is the account that will receive reward'
```
image

File: packages/app-staking/src/Overview/MyStake/index.tsx (hunk 2 above)
Comment (AlexZhenWang) [reply packages/app-staking/src/Overview/MyStake/index.tsx:1]: Created a ticket in the backlog. IND-479

**Comment by AlexZhenWang** [comment]
Created a ticket for updating license: IND-479

**Review by holygits — COMMENTED** [review]
Looks good!
The staking queries are quite complex, I wonder if we can move them to uncover

Requesting some small design changes:
- adjust padding like so
- border radius on the table
- vertical align the text in the table rows
Screen Shot 2021-01-25 at 12 21 45

**Comment by AlexZhenWang** [comment]
> Looks good!
> The staking queries are quite complex, I wonder if we can move them to uncover in the future.
> 
> Requesting some small design changes:
> 
> * adjust padding like so
> * border radius on the table
> * vertical align the text in the table rows
> 
> Screen Shot 2021-01-25 at 12 21 45

Yeah, updated the style
image

**Comment by AlexZhenWang** [comment]
> Looks good!
> The staking queries are quite complex, I wonder if we can move them to uncover in the future.
> 
> Requesting some small design changes:
> 
> * adjust padding like so
> * border radius on the table
> * vertical align the text in the table rows
> 
> Screen Shot 2021-01-25 at 12 21 45

We can move the request to uncover but that will make uncover to be a dependency of cennznet/ui

**Comment by AlexZhenWang** [comment]
Update feedback from @holygits :
Add spinner when the page loading

**Comment by AlexZhenWang** [comment]
image
Updated style: 
1. Adjust padding
2. Vertical align the text in the table rows

**Review by holygits — APPROVED** [review]
lgtm!

**Pull request merged**